addons/l10n_pe/models/template_pe.py

```diff
@@ -35,3 +35,26 @@ def _get_pe_res_company(self):
                 'account_journal_early_pay_discount_gain_account_id': 'chart775',
             },
         }
+
+    @template(model='account.journal')
+    def _get_account_journal(self, template_code):
+        result = super()._get_account_journal(template_code)
+        if template_code != "pe":
+            return result
+        result.update({
+            "pe_opening": {
+                "name": _("Opening Operations"),
+                "type": "general",
+                "code": _("OPOP"),
+                "show_on_dashboard": True,
+                "sequence": 20,
+            },
+            "pe_closing": {
+                "name": _("Closing Operations"),
+                "type": "general",
+                "code": _("CLOP"),
+                "show_on_dashboard": True,
+                "sequence": 21,
+            },
+        })
+        return result
```

Comment:
Wouldn't something like this work instead of checking the `template_code` ?
https://github.com/odoo/odoo/blob/saas-16.4/addons/l10n_mx/models/template_mx.py#L42